keps/sig-storage/1710-selinux-relabeling/kep.yaml

```diff
@@ -0,0 +1,35 @@
+title: Skip SELinux relabeling of volumes
+kep-number: 1710
+authors:
+  - "@jsafrane"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-auth
```

Comment:
nit: I don't think this KEP is relevant to sig-auth.

Reply:
removed